recommender/recommender.py

```diff
@@ -968,6 +970,23 @@ def studio_view(self, _context=None):  # pylint: disable=unused-argument
         frag.initialize_js('RecommenderXBlock')
         return frag
 
+    def add_xml_to_node(self, node):
+        """
+        Serialize the XBlock to XML for exporting.
+        """
+        node.tag = 'recommender'
+
+        node.set('intro_enabled', str(self.intro_enabled))
+        node.set('DISABLE_DEV_UX', str(self.client_side_settings['DISABLE_DEV_UX']))
+        node.set('ENTRIES_PER_PAGE', str(self.client_side_settings['ENTRIES_PER_PAGE']))
+        node.set('PAGE_SPAN', str(self.client_side_settings['PAGE_SPAN']))
```

Comment:
I can't find a way to serialize integer and boolean properly. It seems to me most people use string for them.
http://stackoverflow.com/questions/19534288/can-xml-etree-elementtree-write-integer-values-for-a-given-element
http://stackoverflow.com/questions/715417/converting-from-a-string-to-boolean-in-python
Is there any way to do that?